src/Controller/ArticlesController.php

```diff
@@ -878,14 +878,10 @@ private function articlePageArguments(Request $request, string $id, int $version
 
                 $exampleRdsArticles = $this->getParameter('example_rds_articles');
                 $rdsArticles = $this->getParameter('rds_articles');
-                $rdsInfoBarUrl = '';
                 if (isset($rdsArticles[$item->getId()]) && $this->isGranted('FEATURE_RDS')) {
-                    $rdsInfoBarUrl = $this->get('router')->generate('article-rds', [$item]);
+                    $infoBars[] = new InfoBar('See this research in an <a href="'.$this->get('router')->generate('article-rds', [$item]).'">executable code view</a>.', InfoBar::TYPE_WARNING);
                 } elseif (isset($exampleRdsArticles[$item->getId()])) {
-                    $rdsInfoBarUrl = $exampleRdsArticles[$item->getId()];
-                }
-                if (strlen($rdsInfoBarUrl) > 0) {
-                    $infoBars[] = new InfoBar('See this research in an <a href="'.$rdsInfoBarUrl.'">executable code view</a>.', InfoBar::TYPE_WARNING);
+                    $infoBars[] = new InfoBar('This research is available in a <a href="'.$exampleRdsArticles[$item->getId()].'">reproducible view</a>.', InfoBar::TYPE_WARNING);
```

Comment:
Sorry, wasn't clear enough. Just meant to open a quick PR to update the wording for the current implementation to reduce the diff here. (Assuming they are meant to be the same.)